<launch>
    <!-- Run node -->
    <node name="recognize_rcv" pkg="slam" type="recognize" output="screen"/>
</launch>